control-plane-operator/hostedclusterconfigoperator/controllers/resources/konnectivity/reconcile.go

```diff
@@ -52,6 +52,7 @@ func ReconcileAgentDaemonSet(daemonset *appsv1.DaemonSet, deploymentConfig confi
 				Labels: labels,
 			},
 			Spec: corev1.PodSpec{
+				DNSPolicy:                    corev1.DNSDefault,
```

Comment:
Please remove below since the change here makes it apply to all providers.

```
daemonset.Spec.Template.Spec.DNSPolicy = corev1.DNSDefault
```

Reply:
Wouldn't be useful for all provider?

Reply:
Yes, but other providers are already doing this.  I'm just saying that there's no need to reset it later in the code.

Reply:
Got it now, thanks!